src/main/java/edu/garyMooradian/advancedJava/model/StockQuote.java

```diff
@@ -0,0 +1,116 @@
+package edu.garyMooradian.advancedJava.model;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Calendar;
+import java.util.Date;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import org.joda.time.DateTime;
+
+
+@Entity
+//@Table(name="quotes2")//original line
+@Table(name="quotes3")//new line
+public class StockQuote {
+	
+	private int id;
+	private String symbol;
+	
+	//private DateTime time;//original line
+	private String time;//new line
```

Comment:
should be able to use a time object here, but I understand from email that you had trouble with this. as a result this is an OK workaround.